src/main/java/net/tardis/mod/common/dimensions/gallifrey/WorldProviderGallifrey.java

```diff
@@ -34,11 +34,11 @@ public IRenderHandler getSkyRenderer() {
 		return SKY;
 	}
 	
-	@Nullable
-	@Override
-	public IRenderHandler getCloudRenderer() {
-		return new RenderVoid();
-	}
+	//@Nullable
```

Comment:
Just remove it entirely, don't bother with commenting this out if we're keeping clouds